### PR TITLE
ci: speedup Rocky Linux build

### PR DESCRIPTION
**Pull request by coryan**
Same problem as Fedora, the default `pkgconfig` tool is slow with some
Abseil libraries.

---
This change is [Reviewable](https://reviewable.io/reviews/googleapis/google-cloud-cpp/7651)

### CODE REVIEW

File: doc/packaging.md [hunk 1]
```diff
@@ -1198,8 +1198,24 @@ sudo dnf makecache && \
...
 sudo dnf makecache && \
-sudo dnf install -y ccache cmake gcc-c++ git make openssl-devel patch pkgconfig \
-        re2-devel zlib-devel libcurl-devel c-ares-devel tar wget which
+sudo dnf install -y ccache cmake curl findutils gcc-c++ git make openssl-devel \
+        patch re2-devel zlib-devel libcurl-devel c-ares-devel tar wget which
+```
+
+Rocky Linux's version of `pkg-config` (https://github.com/pkgconf/pkgconf) is
+slow when handling `.pc` files with lots of `Requires:` deps, which happens
+with  Abseil. If you plan to use `pkg-config` with any of the installed
```
Comment (devbww) [inline doc/packaging.md:1207]: Nit: two-space gap -> one.

File: ci/cloudbuild/dockerfiles/demo-rockylinux-8.Dockerfile [hunk 2]
```diff
@@ -25,8 +25,24 @@ RUN dnf makecache && \
...
     dnf makecache && \
-    dnf install -y ccache cmake gcc-c++ git make openssl-devel patch pkgconfig \
-        re2-devel zlib-devel libcurl-devel c-ares-devel tar wget which
+    dnf install -y ccache cmake curl findutils gcc-c++ git make openssl-devel \
+        patch re2-devel zlib-devel libcurl-devel c-ares-devel tar wget which
+# ```
+
+# Rocky Linux's version of `pkg-config` (https://github.com/pkgconf/pkgconf) is
+# slow when handling `.pc` files with lots of `Requires:` deps, which happens
+# with  Abseil. If you plan to use `pkg-config` with any of the installed
```
Comment (devbww) [inline ci/cloudbuild/dockerfiles/demo-rockylinux-8.Dockerfile:34]: Nit: two-space gap -> one.

**Review by devbww — APPROVED** [review]

File: ci/cloudbuild/dockerfiles/demo-rockylinux-8.Dockerfile (hunk 2 above)
Comment (coryan) [reply ci/cloudbuild/dockerfiles/demo-rockylinux-8.Dockerfile]: Fixed.

File: doc/packaging.md (hunk 1 above)
Comment (coryan) [reply doc/packaging.md]: Fixed.

**Pull request merged**